README.md

```diff
@@ -8,7 +8,7 @@
 
 TPI is a [Terraform](https://terraform.io) plugin built with machine learning in mind. This CLI tool offers full lifecycle management of computing resources (including GPUs and respawning spot instances) from several cloud vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
 
-- **Lower cost with spot recovery**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **Lower cost with spot recovery**: transparent data backup & auto-recovery from interrupted low-cost spot/preemptible instances
```

Comment:
```suggestion
- **Lower cost with spot recovery**: transparent data checkpoint & restore from interrupted low-cost spot/preemptible instances
```

Traditionally “backup” has a long–term connotation that may be misleading. 🤔 Not that my suggested change is much clearer `:muk:` but probably conveys better the functionality.

Reply:
```suggestion
- **Lower cost with spot recovery**: transparent data checkpoint/restore & auto-respawning of low-cost spot/preemptible instances
```